src/parsing.jl

```diff
@@ -252,7 +252,7 @@ end
 ## arrays
 # numeric arrays never have double quotes and always use ',' as a separator
 function parse_numeric_array(eltype::Type{T}, str::AbstractString) where T
-    eq_ind = searchindex(str, '=')
+    eq_ind = something(findfirst(isequal('='), str), 0)
```

Comment:
Can you just change the `eq_ind > 0` check to `eq_ind !== nothing` and avoid the `something`?

Reply:
On 0.6, this returns 0 rather than `nothing` when there's no match. `something` is just a more concise way to handle both versions at once.

Reply:
Ohhh, so this is for cross-version support. Maybe make a comment showing that we can change it when we drop 0.6